### PR TITLE
Add a usage guide to the README.md of the Twitter Source.

### LINKED ISSUE #40: Code sample for twitter
The Twitter connector readme doesn't show how to use the connector.

Please, link the https://github.com/hazelcast/hazelcast-jet-demos/tree/master/cryptocurrency-sentiment-analysis/ or provide a code snippet in the readme.

### PR DESCRIPTION
**Pull request by ufukyilmaz**
### What this PR does / why do we need it:
This PR updates the README.md of the Twitter Source. I added a usage guide to the Twitter Source.
#### Which issue this PR fixes
  - It fixes #40 
         
#### Checklist

- [x] Signed the Hazelcast CLA
- [x] No checkstyle issues (`./gradlew check`)
- [x] No tests failures (`./gradlew test`)
- [x] Documentation which complies with README template (see `templates/README.template.md`).
- [x] Update `List of modules` section on the root README with your module.

### CODE REVIEW

File: twitter/README.md [hunk 1]
```diff
@@ -55,6 +54,66 @@ To run the tests run the command below:
...
+properties.setProperty("token", "???"); // OAuth1 Token
+properties.setProperty("tokenSecret", "???"); // OAuth1 Token Secret
+List<String> terms = new ArrayList<>(Arrays.asList("BTC", "ETH"));
+StreamSource<String> streamSource =
+             TwitterSources.stream(
+                     credentials,
+                     () -> new StatusesFilterEndpoint().trackTerms(terms)
+             );
+Pipeline p = Pipeline.create();
+StreamSourceStage<String> srcStage = p.readFrom(streamSource);
```
Comment (eminn) [inline twitter/README.md]: Can you put a complete pipeline which writes to the logger sink ?

File: twitter/README.md [hunk 2]
```diff
@@ -55,6 +54,66 @@ To run the tests run the command below:
...
+rate-limit is exceeded.
+
+Here is an example which searches with the query `Jet flies`.
+
+```java
+Properties credentials = loadCredentials(); // Load Twitter OAuth1 credentials
+String query = "Jet flies";
+BatchSource<Status> searchSource = TwitterSources.search(credentials, query);
+Pipeline pipeline = Pipeline.create();
+BatchStage<Status> srcStage = pipeline.readFrom(searchSource)
```
Comment (eminn) [inline twitter/README.md]: Can you put a complete pipeline which writes to the logger sink ?

File: twitter/README.md [hunk 3]
```diff
@@ -55,6 +54,66 @@ To run the tests run the command below:
...
+Connector performs multiple requests for the same query to get all available
+pages. Twitter has a rate limit for the search endpoint that limits the number
+of these search requests to 180 calls every 15 minutes. Twitter Connector
+repeatedly makes search requests until it gets all available pages or the
+rate-limit is exceeded.
+
+Here is an example which searches with the query `Jet flies`.
+
+```java
+Properties credentials = loadCredentials(); // Load Twitter OAuth1 credentials
```
Comment (eminn) [inline twitter/README.md]: Could you please make this consistent with the way properties provided in the example above ?

Comment (ufukyilmaz) [reply twitter/README.md]: I have changed it to the way you say.

File: twitter/README.md (hunk 1 above)
Comment (ufukyilmaz) [reply twitter/README.md]: I have updated the examples with respect to your review.

File: twitter/README.md (hunk 2 above)
Comment (ufukyilmaz) [reply twitter/README.md]: I have updated the examples with respect to your review.

File: twitter/README.md [hunk 4]
```diff
@@ -55,6 +54,88 @@ To run the tests run the command below:
...
+properties.setProperty("tokenSecret", "???"); // OAuth1 Token Secret
+List<String> terms = new ArrayList<>(Arrays.asList("BTC", "ETH"));
+StreamSource<String> streamSource =
+             TwitterSources.stream(
+                     credentials,
+                     () -> new StatusesFilterEndpoint().trackTerms(terms)
+             );
+Pipeline p = Pipeline.create();
+p.readFrom(streamSource)
+        .withoutTimestamps()
```
Comment (eminn) [inline twitter/README.md]: Can you align chained method calls around the dot ?

File: twitter/README.md [hunk 5]
```diff
@@ -55,6 +54,88 @@ To run the tests run the command below:
...
+Properties credentials = new Properties();
+properties.setProperty("consumerKey", "???"); // OAuth1 Consumer Key
+properties.setProperty("consumerSecret", "???"); // OAuth1 Consumer Secret
+properties.setProperty("token", "???"); // OAuth1 Token
+properties.setProperty("tokenSecret", "???"); // OAuth1 Token Secret
+String query = "Jet flies";
+BatchSource<Status> searchSource = TwitterSources.search(credentials, query);
+Pipeline p = Pipeline.create();
+p.readFrom(searchSource)
+        .map(status -> "@" + status.getUser().getName() + " - " + status.getText())
```
Comment (eminn) [inline twitter/README.md]: Can you align chained method calls around the dot ?

Comment (ufukyilmaz) [reply twitter/README.md]: I aligned them.

File: twitter/README.md (hunk 4 above)
Comment (ufukyilmaz) [reply twitter/README.md]: I aligned them.

**Comment by eminn** [comment]
LGTM

**Pull request merged**